### ISSUE TITLE
Dol Guldur - Rebuild / cleanse Dol Guldur

### ISSUE THREAD

**Issue by jj248** [MattTheLegoman/RealmsInExile#542]
Mechanic to build up and or cleanse Dol Guldur.

This could be done with decisions, or just through upgrading the special building.

Can we replace the building with an alternate good version by effect to allow both to be upgradable?

**Comment by Jauffr** [comment]
Dol Guldur special building is functional and working as intended. The building doesn't do really anything and is useless to build for the cost, however it does in fact work
![Screenshot 2023-02-28 182634](https://user-images.githubusercontent.com/108106660/222021375-8d68ebd2-e531-41c3-9ff9-a078a0772368.png)

**Comment by wFredsch** [comment]
Is it the amon lanc renewed?

Verzonden vanaf Outlook voor Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by wFredsch** [comment]
What do you suggest for it?

Verzonden vanaf Outlook voor Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by RedArkady** [comment]
Yes. I keep pleading for people to advise on what these modifiers should be rather than just complaining about them! Easy enough to fix, just let me know your advice and justification.

**Comment by savagesach** [comment]
so I've thought about what modifiers the amon lanc renewed building should have, and here is my idea.
this holding: fort level +2
this county: development growth +0.20/month, holding taxes +20%
holder of the holding: same culture opinion +10

**Comment by wFredsch** [comment]
Seems fair, if red or vector is oké with it I'll implement it.

Verzonden vanaf Outlook voor Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by savagesach** [comment]
the bonuses for the amon lanc renewed building work as intended, however I think most if not all of the bonuses should be added to amon lanc as well for continuity purposes.
![image](https://user-images.githubusercontent.com/86676697/227321440-ec86e936-cd26-4f40-a223-d2339659b095.png)
![image](https://user-images.githubusercontent.com/86676697/227321488-1d376fda-0626-4b17-a157-1ed56188179c.png)
